mypy/build.py

```diff
@@ -127,12 +127,20 @@ def build(sources: List[BuildSource],
         directories; if omitted, use '.' as the data directory
     """
 
-    data_dir = default_data_dir(bin_dir)
+    if options.custom_typeshed_dir:
+        custom_data_dir, custom_typeshed_name = os.path.split(
+            options.custom_typeshed_dir.rstrip(os.sep))
+        data_dir = custom_data_dir
+    else:
+        custom_typeshed_name = None
```

Comment:
Actually, this got changed in a different way from the refactor to fix the other issues.